_posts/2021-02-25-paul-richmond.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Working with you rather than for you"
+author: "@pweschmidt"
+rse: "Paul Richmond"
+excerpt: "Paul Richmond has stepped up to be the new president of the Society for Research Software Engineering in the UK. After years of masquerading as researcher he was finally able to move into a role as RSE. Oh and watch out for what's hidden in his garden."
+date: 2021-02-25 8:30:00
+external_media: https://us-rse.org/rse-stories-episodes-1/2021/rse-stories-paul-richmond-episode-53.mp3
+length: 14163927
+duration: "00:29:30"
+explicit: "no"
+resources:
+  - name: Paul Richmond Twitter
+    url: https://twitter.com/gpu_mondus 
+  - name: Paul Richmond Sheffield University, UK
+    url: http://paulrichmond.shef.ac.uk
+  - name: Paul Richmond Society for Research Software Engineering, UK
+    url: https://rse.ac.uk/paul-richmond/ 
+  - name: FLAME GPU github
+    url: https://github.com/FLAMEGPU 
+--- 
+
+Now that research software engineering is an established role in some universities and countries it is easy to forget how difficult it was for engineers to work on research they love but really focus on the software development side rather than the academic research. In this interview [Paul Richmond](http://paulrichmond.shef.ac.uk) describes how he "masqueraded" as academic before he got the fellowship at the [Engineering and Physical Sciences Research Council - EPSRC](https://epsrc.ukri.org) in the UK. Since then he established himself firmly as an RSE and built a team of engineers at the university of Sheffield in the UK. 
```

Comment:
I think this is still the case - it's harder to get funding just for an engineering / rse project than traditional science or research. But it definitely varies depending on the country / institution you are at!

Reply:
This is definitely my phenotype too, heh.